templates/repo/issue/list.tmpl

```diff
@@ -111,81 +118,16 @@
 				</div>
 			</div>
 		</div>
-		<div id="issue-actions" class="ui stackable grid">
```

Comment:
Why this block is removed?

Reply:
This block was what was used to replace the above block when the buttons swapped. 

The first block
![gitea issues](https://user-images.githubusercontent.com/42128690/52671205-0d2b3c00-2ee0-11e9-8b78-43d020a38270.png)

Were replaced by the actions
![gitea issue actions](https://user-images.githubusercontent.com/42128690/52671214-14eae080-2ee0-11e9-8571-4142dac119e0.png)

Now there is just a button next to the issue block that shows up when they start selecting issues.

Reply:
I see there were fewer sort option for the second block, is that something I need to revert? The page reloads when one is selected anyways, so there wouldn't be any conflicts that I can think of to leave all filters.

Reply:
Wow, I apparently completely misinterpreted what that second block actually did. My apologies, I will work it back in.